src/superphot_plus/fit_numpyro.py

```diff
@@ -100,18 +103,27 @@ def run_mcmc(lc, sampler="NUTS", t0_lim=None, plot=False):
     rng_key = random.PRNGKey(4)
```

Comment:
I know it is not part of this PR, but should this be a fixed random seed? I would assume that, unless you are testing, you'd want different random values from the runs.

Reply:
Yeah it's there for reproducibility, but I'll probably move it to the tests at one point.